### ISSUE TITLE
[Feature] Add Badges to README

### ISSUE THREAD

**Issue by kindoflew** [fastruby/next_rails#27]
`next_rails` is FastRuby's 2nd most starred project (after Skunk) so it would be nice to have badges in the README like the other projects do. 

**Before we start...:**

- [x] I checked the documentation and didn't find this feature
- [x] I checked to make sure that this feature has not already been requested

**Problem:**

> **As a potential user of `next_rails`**
> **I want to see relevant stats about the project**
> **So that I can accurately assess if I'd like to use it**

**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**
